Packages/vcs/Lib/VTKPlots.py

```diff
@@ -458,6 +465,7 @@ def isopened(self):
 
     def geometry(self, x, y, *args):
         self.renWin.SetSize(x, y)
+        self._lastSize = (x, y)
```

Comment:
I know in the past there was an issue with vistrails. Is that still true? I would rather prefer not to have this convoluted logic behind previous and the current size if possible.